tenbilac/com.py

```diff
@@ -400,6 +462,15 @@ def predict(self, inputs):
 		
 
 
+def datetimestr(dt):
```

Comment:
Yes, but com.py is the only "interface" of Tenbilac that will ever care about such things, so I leave this one here.